gazu/asset.py

```diff
@@ -146,11 +146,12 @@ def new_asset(
     """
     project = normalize_model_parameter(project)
     asset_type = normalize_model_parameter(asset_type)
+    episode = normalize_model_parameter(episode)
 
     data = {"name": name, "description": description, "data": extra_data}
 
-    if episode_id is not None:
-        data["episode_id"] = episode_id
+    if episode is not None:
+        data["episode_id"] = episode["episode_id"]
```

Comment:
The problem is here: `episode["id"]` is expected. Sorry I read too fast during the first review.